### PR TITLE
Changed course card to collapse

### PR DESCRIPTION
**Pull request by martinezmarina**
# What's here (& why)

- (include link(s) to applicable Trello card(s))

# Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete / work-in-progress, PR is for discussion or feedback
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

- [ ] Tested locally
- [ ] Test suite provides the same results as prior to making changes

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/RestModules/ModuleCard.js [hunk 1]
```diff
@@ -1,15 +1,36 @@
 import React from 'react';
-import { Card } from 'antd';
+// import { Card } from 'antd';
```
Comment (chazkiker2) [inline src/components/RestModules/ModuleCard.js:2]: delete commented-out code

File: src/components/RestModules/ModuleCard.js [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+    },
+  ];
   return (
     <>
-      <Card title={module.modulename}>
-        <h3>Description: {module.moduledescription}</h3>
-        {/* PLEASE DO NOT REMOVE THE FOLLOWING LINE */}
-        {props.children}
-      </Card>
+      {/* <Card title={module.modulename}>
```
Comment (chazkiker2) [inline src/components/RestModules/ModuleCard.js:28]: delete commented-out code

File: .env.sample [hunk 3]
```diff
@@ -1,3 +0,0 @@
-REACT_APP_CLIENT_ID=example
```
Comment (chazkiker2) [inline .env.sample:1]: the reason there is a check failing is because you deleted this file... please add this file back into your root folder

**Review by chazkiker2 — CHANGES_REQUESTED** [review]
if you add `.env.sample` back into your project and delete the commented-out code, this will be good to go.

**Review by sebass482 — CHANGES_REQUESTED** [review]
Great changes, some edits needed. 1 check failing.

**Pull request closed without merging**